src/node_v8.cc

```diff
@@ -243,6 +248,39 @@ void SetFlagsFromString(const FunctionCallbackInfo<Value>& args) {
   V8::SetFlagsFromString(*flags, static_cast<size_t>(flags.length()));
 }
 
+void StartCpuProfile(const FunctionCallbackInfo<Value>& args) {
+  Environment* env = Environment::GetCurrent(args);
+  Isolate* isolate = env->isolate();
+  CpuProfilingResult result = env->StartCpuProfile();
+  if (result.status == CpuProfilingStatus::kErrorTooManyProfilers) {
+    return THROW_ERR_CPU_PROFILE_TOO_MANY(isolate,
+                                          "There are too many CPU profiles");
+  } else if (result.status == CpuProfilingStatus::kStarted) {
+    args.GetReturnValue().Set(Number::New(isolate, result.id));
+  }
+}
+
+void StopCpuProfile(const FunctionCallbackInfo<Value>& args) {
+  Environment* env = Environment::GetCurrent(args);
+  Isolate* isolate = env->isolate();
+  CHECK(args[0]->IsUint32());
+  uint32_t profile_id = args[0]->Uint32Value(env->context()).FromJust();
+  CpuProfile* profile = env->StopCpuProfile(profile_id);
+  if (!profile) {
+    return THROW_ERR_CPU_PROFILE_NOT_STARTED(isolate,
+                                             "CPU profile not started");
+  }
+  auto json_out_stream = std::make_unique<node::JSONOutputStream>();
+  profile->Serialize(json_out_stream.get(),
+                     CpuProfile::SerializationFormat::kJSON);
```

Comment:
Generating a string is likely fine for now. Later, I think it might be interesting to allow the cpu profile to be streamed out using a `stream.Readable` or `ReadableStream`

Reply:
OK, we can implement this in CpuProfileHandle in the future.